src/helpers.ts

```diff
@@ -10,25 +10,31 @@ import {
 
 import { tryExtensions, findUp, cjsRequire, isPkgAvailable } from '@pkgr/core'
 
-import { compareVersion } from './common.js'
```

Comment:
@coderabbitai generate more unit testing code for this file.

Reply:
@coderabbitai The tests run in ESM only environment, so there are a lot of cases with `require` are invalid.

Reply:
@coderabbitai ESM modules can not be mocked like this which is absolutely wrong:

```ts
jest.spyOn(isPkgAvailable, 'isPkgAvailable').mockReturnValue(true)
```

Reply:
@coderabbitai `process.stdout` is readonly, so can not be mocked like that.